tools/release/create_notes.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
The package should be called `tool` (singular) - we prefer singular to plural when naming packages

Reply:
This folder is more akin to "dependencies" and "tests" than it is to "util" or "test". We have a mild precedent with "resources" in Studio and "templates" in Console. I'm inclined to stick with "tools".

Reply:
You should stick with `tool` since the the package you're pulling from is called `//.../tool`. Let's be consistent for now and revisit plurasation discussion at another date.

Reply:
Separate note: only python and rust call it `tests`, while Node and Java use `test`

Reply:
@dmitrii-ubskii once you rename this I can merge it